AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminOpenAdobeStockSearchModalActionGroup.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminOpenAdobeStockSearchModalViaGalleryButtonActionGroup">
```

Comment:
Just for note, we need to make sure that we are not duplicating the action groups with different names but same actions. Please check the action group introduced here https://github.com/magento/adobe-stock-integration/pull/239/files#diff-f9da93ddc7a90bd437d0174ab9c8ded2R11

Reply:
@rogyar I've checked this pull request yesterday, it different action group with different behavior.